packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bsmi ocaml library"
+description: "A very useful tool collections"
+maintainer: ["cnmade"]
+authors: ["cnmade"]
+license: "AGPL-3.0-only"
+tags: ["topics" "tools" "your" "project"]
+homepage: "https://github.com/infobsmi/bsmi_ocaml"
+doc: "https://url/to/documentation"
```

Comment:
This field is optional. If you don’t have documentation you can just remove this line